lib/App/Prove.pm

```diff
@@ -772,17 +771,6 @@ along with a reference to the C<App::Prove> object that is invoking your plugin:
       ...
   }
 
-Note that the user's arguments are also passed to your plugin's C<import()>
-function as a list, eg:
```

Comment:
Maybe mention that this used to be the case until whatever version this will be in, in case DarkPAN users get bitten by this. Also it should be mentioned in `Changes`.